### DISCUSSION TITLE
Figure out how to show extended excellent documentation

### DISCUSSION THREAD

**Discussion by nicpottier** [nyaruka/floweditor#1113] [Ideas]
We should either link off to the docs for the function somehow or have a way of expanding the help when writing expressions.

Has some questions on how `format_datetime` worked and full docs answered it easily but couldn't do it without.

**Comment by rowanseymour** [comment]
Yeah think this would be a huge help for users

In Excel you can click on the function name to get the docs in a popup:

<img width="429" alt="Captura de Pantalla 2019-08-22 a la(s) 09 21 11" src="https://user-images.githubusercontent.com/675558/63522607-54613d80-c4be-11e9-8089-01035f210dd8.png">
